### ISSUE TITLE
paper-select : parentElement is null error after using

### ISSUE THREAD

**Issue by larryh** [miguelcobain/ember-paper#1139]
Hi,

First of all: thanks for all of the work you folks do on Ember Paper!

I just upgraded ember from 3.15 to 3.17, and ember-paper from .beta.26 to .beta.29 and am having a problem with ember-select.

The drop-down has a list of states, and when you select a state you get a list of the politicians in that state. It starts out ok, and when you select a state you get the correct list of pols, but in so doing you get a `parentElement is null` error (which you can see in the console) and the app's entire UI is frozen.

Also note that in the last screenshot, the selected state, e.g. Massachusetts, does not appear in the select-box, instead the "Select A State" placeholder has "moved up" above the control and stayed there.

I have attached screenshots of the process (in order, i.e. 1_xxx, 2_yyy, 3_zzz). Here is the code that I use, which has worked for a long time - even with the new "Octane Way" of doing things.

Can someone shed some light on this? Obviously I had to back down to .beta.26 so I can use the app. 

Actually, I guess I should try moving to .27 and then .28 to see exactly where this happens. I will do that and update this issue with my findings. In the meantime, here is the code that I use (it's a component):

`  <PaperSelect 
data-test-select-pols-state
@label="State"
@placeholder="Select a State"
@options={{this.states}}
@selected={{this.selectedState}}
@onChange={{fn this.filter "state"}}
 class="column-control" as |state| >
    {{state.name}} 
<\/PaperSelect>

`

![1_InitialState](https://user-images.githubusercontent.com/89646/77232472-1e2c8f00-6b6f-11ea-9158-7dfc7c4283d5.png)
![2_States](https://user-images.githubusercontent.com/89646/77232473-1ec52580-6b6f-11ea-9012-e007fbe1b418.png)
![3_AfterSelection](https://user-images.githubusercontent.com/89646/77232474-1ec52580-6b6f-11ea-82aa-a61b721fef5a.png)

**Comment by larryh** [comment]
Hello again,

Turns out the problem was introduced in .beta.27. Hope this helps.

**Comment by Gaelan** [comment]
Running into this with `paper-menu` as well.

**Comment by ctwoolsey** [comment]
I've run into this as well with paper-select, but when I tried using beta.27, when I try ember serve, though, I get a message saying, "Could not find the default SASS implementation."  I've tried using different versions of ember-cli-sass (8.0.1, 9.0.1, 10.0.0 & 10.0.1) but this did not get rid of the error. Going back to v29 gets rid of this.   @larryh did you run into this?

**Comment by larryh** [comment]
I can't remember, as it was awhile ago. It does sound familiar, though. (Which probably doesn't help much.)

**Comment by ctwoolsey** [comment]
@larryh could you share your package file that is using beta26?

**Comment by larryh** [comment]
Sure. (Had to slap a ".txt" on the end because would not allow ".json" files to be attached)

[package.json.txt](https://github.com/miguelcobain/ember-paper/files/4438860/package.json.txt)

**Comment by ctwoolsey** [comment]
@larryh Thank you.  beta26 got rid of the "parentElement is null" error.  And I found I hadn't named a default implementation of Sass.

**Comment by larryh** [comment]
Glad I could be of help.

**Comment by shibulijack** [comment]
I faced the same issue with `paper-autocomplete` and the fix was to downgrade to **.beta26** and implement **Node SASS** as the default implementation as Dart Sass resulted in an error with angular's mixin.

However, it'd be nice if this issue is fixed. Keep up the amazing work! :)

**Comment by alex9nadon** [comment]
It seem to be related to https://github.com/cibernox/ember-basic-dropdown/issues/540

**Comment by miguelcobain** [comment]
Can someone check on beta.30?
I changed something that might have fixed this.

**Comment by alex9nadon** [comment]
It works on my side (beta.30 & ember 3.17)
`@animationEnabled={{false}}` On ember-basic-dropdown seem to fix the problem.

**Comment by ctwoolsey** [comment]
I'm still getting the null errors.  Do I have to change @animationEnabled={{false}} in order for it to work?  If so where do i put it.  I tried adding it to a <PaperSelect> but that did not fix the issue either.

**Comment by miguelcobain** [comment]
@ctwoolsey what @alex9nadon meant was that on beta.30, ember-paper added `@animationEnabled={{false}}`.
what version of ember-basic-dropdown are you using? You can check it with `npm ls ember-basic-dropdown`.

**Comment by ctwoolsey** [comment]
Currently I see that I have version 1.1.3.  This is when I am using beta.26.

**Comment by shibulijack** [comment]
> It works on my side (beta.30 & ember 3.17)
> `@animationEnabled={{false}}` On ember-basic-dropdown seem to fix the problem.

I suppose it fixes the issue partially. There's still a console.log error on `animateOut` during dropdown close, but it's usable since `beta.30`.

**Comment by miguelcobain** [comment]
@ctwoolsey please use beta.30

@shibulijack what console log do you see?

**Comment by ctwoolsey** [comment]
So using beta 30, I see ember-basic-dropdown v 2.0.15
The console error is:

> Uncaught (in promise) TypeError: Cannot read property 'appendChild' of null
    at PaperSelectEbdContent.animateOut (component.js:64)
    at Object.destroyModifier (will-destroy.js:68)
    at CustomModifierState.destroy (index.js:7052)
    at StringDestroyableDestructor.[DID_DROP] (util.js:355)
    at util.js:266
    at Set.forEach (<anonymous>)
    at didDestroyAssociated (util.js:265)
    at SimpleDestructor.[DID_DROP] (util.js:379)
    at util.js:266
    at Set.forEach (<anonymous>)
animateOut @ component.js:64
destroyModifier @ will-destroy.js:68
destroy @ index.js:7052
[DID_DROP] @ util.js:355
(anonymous) @ util.js:266
didDestroyAssociated @ util.js:265
[DID_DROP] @ util.js:379
(anonymous) @ util.js:266
didDestroyAssociated @ util.js:265
[DID_DROP] @ util.js:379
(anonymous) @ util.js:266
didDestroyAssociated @ util.js:265
[DID_DROP] @ util.js:379
(anonymous) @ util.js:266
didDestroyAssociated @ util.js:265
[DID_DROP] @ util.js:379
(anonymous) @ util.js:304
[DID_DROP] @ util.js:304
commit @ runtime.js:1655
commit @ runtime.js:1803
inTransaction @ runtime.js:1961
_renderRoots @ index.js:9043
_renderRootsTransaction @ index.js:9104
_revalidate @ index.js:9146
invoke @ backburner.js:338
flush @ backburner.js:229
flush @ backburner.js:426
_end @ backburner.js:960
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_end @ backburner.js:970
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_end @ backburner.js:970
Backburner._boundAutorunEnd @ backburner.js:629
Promise.then (async)
iterations @ backburner.js:28
flush @ index.js:43
_scheduleAutorun @ backburner.js:1179
_ensureInstance @ backburner.js:1167
ensureInstance @ backburner.js:922
(anonymous) @ index.js:568
dirtyTagFor @ validator.js:518
markObjectAsDirty @ index.js:637
notifyPropertyChange @ index.js:675
set @ index.js:1622
set @ observable.js:176
close @ component.js:57
handleClose @ power-select.js:348
close @ basic-dropdown.js:122
handleRootMouseDown @ basic-dropdown-content.js:160

>

**Comment by fiddler** [comment]
Getting the same error on my end with 1.0.0-beta.30 as @ctwoolsey above using `paper-menu`. Running on `ember-basic-dropdown` @2.0.15.

**Comment by miguelcobain** [comment]
@fiddler @ctwoolsey does running `npm ls ember-basic-dropdown` return you `2.0.15`?

**Comment by fiddler** [comment]
Yes, it returns `ember-basic-dropdown@2.0.15`. You're not seeing it on your end @miguelcobain?

**Comment by ctwoolsey** [comment]
It seems that the animate out error, causes a problem with ember-power-select-multiple.  None of the selections get loaded.  As soon as I downgraded back to beta 26, ember-power-select-multiple was able to work again.

**Comment by fiddler** [comment]
This is no longer happening with 1.0.0-beta.32. Seems this commit here prevents the error: https://github.com/miguelcobain/ember-paper/commit/d1336eef959e3016f696e72210f4327719ad94b4

**Comment by ctwoolsey** [comment]
I am still seeing: "TypeError: Cannot read property 'appendChild' of null"  And, ember-power-select-multiple is not showing my selections, when I try beta.32.

**Comment by rahsheen** [comment]
> This is no longer happening with 1.0.0-beta.32. Seems this commit here prevents the error: [d1336ee](https://github.com/miguelcobain/ember-paper/commit/d1336eef959e3016f696e72210f4327719ad94b4)

Definitely still seeing this on beta32

**Comment by wukongrita** [comment]
![image](https://user-images.githubusercontent.com/25361005/98046296-890bab80-1e5c-11eb-9545-8bc36f0fa869.png)

Upgrading to ember 3.20 with beta.35 and still get this when the dropdown closes...

Not happening on 3.16 with beta.35

**Comment by ctwoolsey** [comment]
Wondering if anyone has had any luck resolving this.  I'm wanting to upgrade my ember version past 3.18 and am seeing deprecation warnings in ember-paper 1.0.0-beta.26.
